tools/mlir-vulkan-runner/CMakeLists.txt

```diff
@@ -0,0 +1,45 @@
+set(LLVM_OPTIONAL_SOURCES
+  VulkanRuntime.cpp
+  )
+
+if (MLIR_VULKAN_RUNNER_ENABLED)
+  message(STATUS "Building the Vulkan runner")
+
+  # At first try "FindVulkan" from:
+  # https://cmake.org/cmake/help/v3.7/module/FindVulkan.html
+  if (NOT CMAKE_VERSION VERSION_LESS 3.7.0)
+    find_package(Vulkan)
+  endif()
```

Comment:
I'd add a else and a proper error message here.

Reply:
I think the error message is at L29. :) The probe is not done yet here.

Reply:
I see that there is a fallback, so throwing error here is not possible indeed, but I still feel that the "you're not using the right CMake version should be something surfaced at some point as a root cause of failure.
Something like adding here: an ```else if ("$ENV{VULKAN_SDK}" STREQUAL "")  message(FATAL_ERROR "Please use at least Make 3.7.0 or provide the VULKAN_SDK path as an environment variable")```